### PR TITLE
Refs [TASK][27162] Add css and i18n

### PR DESCRIPTION
**Pull request by duytu131995**
https://edu-redmine.sun-asterisk.vn/issues/27162
Screenshot from 2020-08-13 16-05-42

### CODE REVIEW

**Comment by duytu131995** [comment]
ready

File: app/assets/stylesheets/flash-devise.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import "color.scss";
```
Comment (saobv-2223) [inline app/assets/stylesheets/flash-devise.scss:1]:
```suggestion
@import 'color';
```

Comment (duytu131995) [reply app/assets/stylesheets/flash-devise.scss]: ok em

**Comment by duytu131995** [comment]
ready

**Pull request merged**